src/main/java/uk/gov/ons/entities/InputAddress.java

```diff
@@ -18,46 +21,68 @@
 public @Data class InputAddress {
 
 	@CsvBindByName(column = "address_line1")
+	@NotBlank(message = "ADDRESS_LINE1 is mandatory")
```

Comment:
I find it difficult to read with out blank line inbetween variables, but up to u. consistency